### PR TITLE
Add Computer Networking

### PR DESCRIPTION
**Pull request by nyquist**
https://github.com/nyquist/awesome-networking

The list includes resources related to computer or data networks, related tools and documentation

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -480,6 +480,7 @@
 - [Software-Defined Networking](https://github.com/sdnds-tw/awesome-sdn)
 - [Network Analysis](https://github.com/briatte/awesome-network-analysis)
 - [PCAPTools](https://github.com/caesar0301/awesome-pcaptools)
+- [Computer Networking](https://github.com/nyquist/awesome-networking)
```
Comment (legodude17) [inline readme.md]: Maybe add a description to the list?

Comment (nyquist) [reply readme.md]: Hi,
I followed the format of the other items in the list. The description is in the list itself, but if needed I can add it.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/nyquist/awesome-networking
✖ Linting

  README.md:4:1
  ✖    1:1   Missing License section                                       remark-lint:awesome/license
  ✖    1:1   Missing or invalid Table of Contents                          remark-lint:awesome/toc
  ✖    4:1   Don’t use multiple top level headings (1:1)                   remark-lint:no-multiple-toplevel-headings
  ✖    5:3   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖    6:3   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖    7:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖    8:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖    9:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   10:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   11:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   12:3   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   13:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   14:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   15:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   16:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   17:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   18:3   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   19:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   20:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   21:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   22:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   23:5   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   26:1   Don’t use multiple top level headings (1:1)                   remark-lint:no-multiple-toplevel-headings
  ✖   33:1   Don’t use multiple top level headings (1:1)                   remark-lint:no-multiple-toplevel-headings
  ✖   60:64  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖   63:1   Don’t use multiple top level headings (1:1)                   remark-lint:no-multiple-toplevel-headings
  ✖   69:54  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖   77:56  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖   82:42  List item description must start with valid casing            remark-lint:awesome/list-item
  ✖   95:52  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖   96:34  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖   97:52  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖  103:56  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖  105:1   Don’t use multiple top level headings (1:1)                   remark-lint:no-multiple-toplevel-headings
  ✖  110:71  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ✖  115:37  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  133:75  List item description must end with proper punctuation        remark-lint:awesome/list-item
```

File: readme.md (hunk 1 above)
Comment (sindresorhus) [reply readme.md]: Yes, should have a description.

**Comment by sindresorhus** [comment]
> You have to review at least 2 other open pull requests. Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**